packages/frontend/src/ee/features/aiCopilot/components/ChatElements/AgentChatBubbles.tsx

```diff
@@ -89,14 +91,23 @@ export const AssistantBubble: FC<{
     isPreview?: boolean;
 }> = memo(({ message, isPreview = false }) => {
     const { agentUuid } = useParams();
-    const { activeProjectUuid } = useActiveProjectUuid();
+    const { projectUuid } = useParams();
 
-    const vizQuery = useAiAgentThreadMessageViz({
+    const queryExecutionHandle = useAiAgentThreadMessageVizQuery({
         agentUuid: agentUuid!,
         message,
-        activeProjectUuid,
+        projectUuid: projectUuid!,
```

Comment:
```suggestion
        projectUuid,
```

hook is only enabled when this is truthy and undefined is also acceptable